### DISCUSSION TITLE
Beta Testing for Frigate?

### DISCUSSION THREAD

**Discussion by dz4news** [blakeblackshear/frigate#4736] [Ask A Question]
Is the beta version of frigate available for testing? If so, how do I access it? I am running Frigate in Docker, in a standalone system.

I know when I was running it in HA Supervised, I could access the beta as an add on... but how do I do that in Docker?

TIA.

**Comment by NickM-27** [comment]
There is currently no beta, https://github.com/blakeblackshear/frigate/pull/4055 is almost ready and will have a beta when the tasks are completed. 

When that happens, simply change your docker tag to switch to the new version.
